org.eclipse.scanning.api/src/org/eclipse/scanning/api/points/models/MultiStepModel.java

```diff
@@ -11,21 +11,33 @@
  *******************************************************************************/
 package org.eclipse.scanning.api.points.models;
 
+import java.beans.PropertyChangeListener;
+import java.beans.PropertyChangeSupport;
 import java.text.MessageFormat;
 import java.util.ArrayList;
 import java.util.List;
 
+import org.eclipse.scanning.api.annotation.ui.FieldDescriptor;
+import org.eclipse.scanning.api.annotation.ui.TypeDescriptor;
+
 /**
  * A model consisting of multiple {@link StepModel}s to be iterated over sequentially.
  * 
  * @author Matthew Dickie
  */
+@TypeDescriptor(editor="org.eclipse.scanning.device.ui.composites.MultiStepComposite")
 public class MultiStepModel extends AbstractPointsModel {
+	
+	private static final String STEPMODELS = "stepModels";
 
 	private List<StepModel> stepModels;
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Rename field "stepModels" to prevent any misunderstanding/clash with field "STEPMODELS" defined on line 31. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1845)